views/partials/prs.hbs

```diff
@@ -1,7 +1,9 @@
 {{#exists prs}}
     <div class="tc white">
         {{#if userImage}}
-            <img class="br-100 w4" height="128" src="{{userImage}}" width="128">
+            <a href = "https://github.com/{{username}}">
```

Comment:
I think we should open this in a new tab/window. You ok to add `target="_blank"`?

Reply:
I'm not super fond of that behavior, I usually middle-click a link for a new tab. If you think it's better then I'll change that!

Reply:
Yeah me neither, just thinking on mobile, if people actually press the face...what do you think?

Reply:
Good point! New tab seems like a fair middleground